### DISCUSSION TITLE
How to update one to Many realtionship data

### DISCUSSION THREAD

**Discussion by sumeet030990** [prisma/prisma#13724] [Q&A]
```prisma
model orders {
  id               Int       @id @default(autoincrement())
  total_case_count Int?
  total_case_cost  Int?
  order_details    details[]
}

model details {
  id           Int       @id @default(autoincrement())
  orders     orders?   @relation(fields: [order_id], references: [id])
  order_id   Int?
  deliver_date DateTime? @db.Date
  quantity     Int?
  stores     stores?   @relation(fields: [store_id], references: [id])
  store_id   Int?
  products   products? @relation(fields: [product_id], references: [id])
  product_id Int?
}
```

During the update, I need to update the values of order & details in both 
There are almost 20, 30 products per order

How can I do that?

**Comment by austincrim** [comment]
Hey @sumeet030990 👋,

Could you describe the kind of update you want more specifically?
What fields are you updating?
Have you tried anything yet?
Could you also share the `products` model?

You could also take a look at our [nested write](https://www.prisma.io/docs/concepts/components/prisma-client/relation-queries#nested-writes) docs, which are usually applicable for updates that touch multiple relations.

**Reply by sumeet030990** [reply]
Hi @austincrim,
I want to update order and order detail data, we are allowing to update the quantities or add new products in the same order. When user is updating the quantity then some entries(total_case_count, total_case_cost) from the order ll be updated, and accordingly in details model entries ll be updated or created as per the order id and the product id 
This is what I want to achieve
and for this I looked into docs, I also checked nested write, but in that, the option for update or updateMany is there but not for dynamic data i.e. I can specify order_id but product Id varies so records should be updated as per the matching product_id

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
